packages/shared/src/hooks/post/useBlockPostPanel.ts

```diff
@@ -142,22 +142,32 @@ export const useBlockPostPanel = (
     [setShowTagsPanel, updateFeedPreferences],
   );
 
+  const onDismissPermanently = useCallback(() => {
+    completeAction(ActionType.HideBlockPanel);
+    setShowTagsPanel({ showTagsPanel: undefined });
+  }, [completeAction, setShowTagsPanel]);
+
   const onClose = useCallback(
-    (forceClose?: boolean) =>
-      setShowTagsPanel({ showTagsPanel: forceClose ? undefined : false }),
-    [setShowTagsPanel],
+    (forceClose?: boolean) => {
+      const isRemovingDownvote = typeof forceClose === 'boolean';
+
+      if (!isRemovingDownvote) {
+        displayToast(getBlockedMessage(0, false), {
+          onUndo: onDismissPermanently,
+          undoCopy: `Don't ask again`,
+        });
```

Comment:
What's the difference between this and the one below now?

Reply:
This will remove the `block panel` forever, the other below will revert the "just blocked" sources/tags